pkg/knuu/instance.go

```diff
@@ -326,6 +343,25 @@ func (i *Instance) AddFile(src string, dest string, chown string) error {
 		}
 		file := k8s.NewFile(dstPath, dest)
 
+		parts := strings.Split(chown, ":")
+
+		if len(parts) != 2 {
+			return fmt.Errorf("invalid format")
+		}
```

Comment:
please provide the comment why 2? 
In addition, make 2 a const variable might help